src/main/windows/SDL_sysmain_runapp.c

```diff
@@ -20,80 +20,104 @@
 */
 #include "SDL_internal.h"
 
-#ifdef SDL_PLATFORM_WIN32
+#ifdef SDL_PLATFORM_WINDOWS
 
+#include "../SDL_runapp.h"
 #include "../../core/windows/SDL_windows.h"
 
-/* Win32-specific SDL_RunApp(), which does most of the SDL_main work,
-  based on SDL_windows_main.c, placed in the public domain by Sam Lantinga  4/13/98 */
```

Comment:
Removed this comment as to not misrepresent the origins of the code since I changed the implementation pretty drastically, but I could restore this if you prefer.